GrindingCity.WebApi/Controllers/BuildingController.cs

```diff
@@ -13,48 +14,56 @@ public class BuildingController : ControllerBase
     {
         // GET all api
         [HttpGet()]
-        public async Task<ActionResult<IEnumerable<Building>>> GetAllBuildings(
-            [FromServices] IBuildingRepository handler)
+        public async Task<ActionResult<GetAllBuildingsResponse>> GetAllBuildings(
+            [FromServices] IBuildingRepository buildingHandler)
         {
-            var result = await handler.GetAllBuildings();
+            var buildings = await buildingHandler.GetAllBuildingsAsync();
+            var result = new GetAllBuildingsResponse(buildings.Select(_ => _.Id));

```

Comment:
We shouldn't use discard operator like this (`_`)

Reply:
done